packages/strapi-plugin-upload/admin/src/containers/ModalStepper/index.js

```diff
@@ -23,7 +23,7 @@ const ModalStepper = ({ isOpen, onToggle }) => {
   useEffect(() => {
     if (currentStep === 'upload' && filesToUploadLength === 0) {
       // Close modal when file uploading is over
-      toggleRef.current();
+      toggleRef.current(true);
```

Comment:
Can you add some comment here to inform that passing true to the `onToggle` props will refetch the data when the modal closes?